app/assets/javascripts/partners.js.coffee

```diff
@@ -0,0 +1,5 @@
+jQuery ->
+  $("#partner_start_date").datepicker({
+    format: "yyyy-mm-dd"
+    autoclose: true
+  })
```

Comment:
Yes very bad, wicked even. Id's should be unique and we are increasing the chance of a collision, especially across views using several partials.
However, I have renamed the file date-picker and it triggers on the css class "date-picker" so now people just have to add "date-picker" in their css class and they get the date-picker for free.